spec/unit/shoulda/matchers/active_record/validate_uniqueness_of_matcher_spec.rb

```diff
@@ -450,9 +450,19 @@
 
         expect(record).to validate_uniqueness
       end
+      context 'when the value has no case sensitivity' do
+        it 'accepts' do
+          model = define_model_validating_uniqueness(
+            attribute_type: :string,
+            validation_options: { case_sensitive: true },
+          )
+          record = create_record_from(model, attribute_name => '123')
+          expect(record).to validate_uniqueness
+        end
+      end
     end
 
-    context 'when case_insensitive is specified' do
+    context 'when case_sensitive is specified' do
```

Comment:
This line shouldn't be changed, "specified" is referring to the matcher. (Really this should say "when qualified with `case_insensitive`", but I'll fix it later.)